### DISCUSSION TITLE
VS Code Viewer Addon

### DISCUSSION THREAD

**Discussion by dacoffey** [microsoft/maker.js#503] [Show and tell]
Hello, 

Thanks everyone for your work on this project, 
I use it all the time to create laser cut acrylic panels for custom electronics cases and enclosures.

I have made a simple VS code addon that will render MakerJS code into SVG, save alongside the JS file, and preview the design in a webview panel. When the source is updated and saved, the changes are shown in the preview window.

https://marketplace.visualstudio.com/items?itemName=COGSMITH.vscode-makerjs

https://github.com/cogsmith/vscode-makerjs

![image](https://user-images.githubusercontent.com/1507389/124372918-2ae9cf00-dc5c-11eb-9774-fe9955360d1d.png)

**Comment by danmarshall** [comment]
Hi @dacoffey - sorry for the delayed response, I was on vacation. Thank you for this really cool contribution!
